packages/v4/src/content/design-guidelines/components/form/forms.md

```diff
@@ -195,12 +195,18 @@ Use progressive disclosure to hide and show fields based on the users’ selecti
  <img src="./img/progressive-disclosure.png" alt="Example of progressive disclosure with one radio button selected to only show those inputs" width="562"/>
 
 ## Errors and validation
-When a form field submission results in an error, let users know as soon as possible. Always present error states on the form using field level errors whenever possible. The error state you can use will depend on whether validation happens on loss of focus or on submission. When an error occurs, any placeholder text will no longer be visible as it will be replaced by the user’s input. Additionally, field level errors will replace any existing helper text until the error is fixed. Therefore, it is important that your error message explains what the user can do to fix it, beyond stating that there is an error.
+When a form field submission results in an error, let users know as soon as possible. Always present error states on the form using field level errors whenever possible. The error state you can use will depend on whether validation happens on loss of focus or on submission. 
+
+Alternatively, you can also choose to emphasize errors using an inline error at the top of the form. Note that the inline error should always be a supplemental component, used *in addition* to field-level errors, and it's message should generally describe the problem.
+
+When an error occurs, any placeholder text will no longer be visible as it will be replaced by the user’s input. Additionally, field level errors will replace any existing helper text until the error is fixed. Therefore, it is important that your error message not only points out the error, but also explains what the user can do to fix it.
 
 For example, instead of simply telling the user there is an “invalid input”,  error messages should specify the correct format to re-write the input in such as “Date format must be MM-YY-DD”.
 
  <img src="./img/error-message.png" alt="Visualized example for right vs. wrong way to write an input error" width="649"/>
 
+ There are two ways to validate errors: on **loss of focus** and **on submission**. The error state you use depends on whether validation happens on the client-side or the server-side.
```

Comment:
This actually repeats what's said in the second sentence at the top. Maybe delete that mention at the top, and leave it here! So delete the sentence that says "The error state you can use will depend on whether validation happens on loss of focus or on submission. "